pkg/kubelet/hyper/hyperclient.go

```diff
@@ -483,23 +487,20 @@ func (client *HyperClient) RemoveImage(imageID string) error {
 }
 
 func (client *HyperClient) RemovePod(podID string) error {
-	v := url.Values{}
-	v.Set(KEY_POD_ID, podID)
-	body, _, err := client.call("DELETE", "/pod?"+v.Encode(), "", nil)
-	if err != nil && !strings.Contains(err.Error(), "Can not find") {
-		return err
-	}
-
-	var result podRemoveResult
-	err = json.Unmarshal(body, &result)
+	conn, err := grpc.Dial(HYPER_SERVER, grpc.WithInsecure())
```

Comment:
Move all connection dial process to `client *HyperClient`, so in these handlers you just exec `cli.PodRemove`